### PR TITLE
Project Complete: feature/structure

### PR DESCRIPTION
**Pull request by Elbie-em**
Please follow the map below to see what we worked on. The requirements for this project can be found [here](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript).
* Set up project with an HTML, CSS and JavaScript files
* Added Linter configurations and files for JavaScript(eslint) and CSS(style lint)
* Built functions that allow players to add marks to a specific cell on the board, and then tie it to the DOM, letting players click on the game board to place their marker.
* Built logic that checks when the game is over by declaring a winner or tie.
* Cleaned up the interface to allow players to put in their names, included a button to start/restart the game and add a display element that congratulates the winning player!
* Added validation in input form.

### CODE REVIEW

File: js/modules/game.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    assignCells();
+    displayPlayerTurn(currentPlayer);
+  };
+
+  return {
+    start, players, selectCell, currentPlayer, changePlayerTurn,
+  };
+};
+
+export { Game }; // eslint-disable-line import/prefer-default-export
```
Comment (maelfosso) [inline js/modules/game.js]: Please, re-enable that configuration and apply a default export.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import { Game } from './modules/game.js'; // eslint-disable-line import/extensions
+
+const Player = (id, name, symbol) => ({ id, name, symbol });
```
Comment (maelfosso) [inline js/index.js]: Move it into `game.js`

Comment (maelfosso) [reply js/index.js]: Make the `game.js` file the module with no code related to DOM.
I mean code with only Logic, controller, etc...
And into `index.js` only DOM manipulation in respect to the output of `game.js`

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @Elbie-em @abdelp 

You did a really nice job.
Nothing really to say except some small changes to do.

Kindly read the comments left below and apply them. Then, submit a new code review request.
Thanks

Mael FOSSO

**Comment by abdelp** [comment]
Fixed all of the issues moving the DOM manipulation to the doman.js module

**Review by whiz25 — APPROVED** [review]
## Approved! :clap: 
You have the requested changes and it's a :+1:  from me. Merge and proceed to next milestone.

### Martin Afani

**Pull request merged**